File: Bai_65/main.py
def check(n):
    s = str(n)
    if s[0] == s[-1]:
        return True
    return False

with open('CIRCLE.INP', 'r') as ifile, open('CIRCLE.OUT', 'w') as ofile:
    a, b = map(int, ifile.readline().split())
    cnt = 0
    for i in range(a, b):
        if check(i):
            cnt += 1
    ofile.write(str(cnt) + '\n')

File: Bai_65/main.cpp
#include<iostream>
#include<fstream>
#define ll long long
using namespace std;

bool check(ll &n){
    string s = to_string(n);
    if(s[0] == s[s.length() - 1])
        return true;
    return false;
}

int main(){
    ifstream ifile("CIRCLE.INP");
    ofstream ofile("CIRCLE.OUT");
    ll a,b; ifile >> a >> b;
    ll cnt = 0;
    for(ll i = a; i < b; i++)
        if(check(i))
            ++cnt;
    ofile << cnt << endl;
}